src/pages/index.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
+import GradeBook from '../GradeBook';
+import { Login, SignUp } from './accounts';
+
+function Pages() {
+	return (
+		<Router>
+			<Switch>
+				<Route path={`/sign-in`}>
+					<Login />
```

Comment:
Maybe keep the names of the route path and the `<Login />` consistent with either sign-in or Login?

Reply:
All changed to SignIn now!